cypress/cypress/e2e/03_submariner_deploy.cy.js

```diff
@@ -22,29 +22,35 @@ describe('submariner - Deployment validation', {
     })
 
     it('test the Submariner add-ons install', { tags: ['deployment'] }, function () {
-        let clusterSetName = Cypress.env('CLUSTERSET')
+        let clusterSetName = Cypress.env('CLUSTERSET')+'11'
         let nattPort = Cypress.env('SUBMARINER_IPSEC_NATT_PORT')
+        let source = Cypress.env('SUBMARINER_CATALOG_SOURCE')
+        let clusters_list = Cypress.env('CLUSTER_LIST')
+        
         clusterSetMethods.createClusterSet(clusterSetName) 
-        submarinerClusterSetMethods.manageClusterSet(clusterSetName)
+        submarinerClusterSetMethods.manageClusterSet(clusters_list, clusterSetName)
 
         cy.get('button').contains('Save').click()
         cy.contains('Submariner add-ons', {timeout: 3000}).should('exist').click()
-        cy.get('#install-submariner').click()
+        cy.wait(500)
```

Comment:
Better to avoid such a "stupid" wait.
Instead to check for condition state we expect to get.